httpx/_types.py

```diff
@@ -107,33 +107,7 @@ def close(self) -> None:
         """
         Subclasses can override this method to release any network resources
         after a request/response cycle is complete.
-
-        Streaming cases should use a `try...finally` block to ensure that
-        the stream `close()` method is always called.
-
-        Example:
-
-            status_code, headers, stream, extensions = transport.handle_request(...)
-            try:
-                ...
-            finally:
-                stream.close()
```

Comment:
This is an example from the old-style Transport API.

We had `.read()` so that users could eg. `stream.read()` here.

The Transport API now just returns fully fledged responses, so this is redundant.